README.md

```diff
@@ -175,12 +175,13 @@ tune download meta-llama/Llama-2-7b-hf \
 --output-dir /tmp/Llama-2-7b-hf \
 --hf-token <HF_TOKEN> \
 ```
+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
 
 ### Llama3 download
 ```bash
 tune download meta-llama/Meta-Llama-3-8B \
 --output-dir /tmp/Meta-Llama-3-8B \
---hf-token <HF_TOKEN> \
+--hf-token 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 \
```

Comment:
Hi there @stefanrattay1  - To avoid malicious actors, you should close this PR and invalidate this token. You can do so by visiting https://huggingface.co/settings/tokens, selecting the "Manage" dropdown and clicking on "Invalidate and refresh". 

If there's something else you're hoping to contribute to torchtune, let us know in the Issues page or feel free to open up another PR! (Just don't add your plaintext HF token :) )